--- /media/hdd/dev/node-addon-api-master/napi-inl.h	2020-01-21 06:41:58.000000000 +0100
+++ node_modules/node-addon-api/napi-inl.h	2020-01-26 16:33:38.000000000 +0100
@@ -3121,8 +3121,10 @@
     Object object = Value();
     // It is not valid to call `napi_remove_wrap()` with an empty `object`.
     // This happens e.g. during garbage collection.
-    if (!object.IsEmpty())
+    if (!object.IsEmpty()) {
       napi_remove_wrap(Env(), object, nullptr);
+      this->SuppressDestruct();
+    }
   }
 }

